.cirrus/tasks.yml

```diff
@@ -48,6 +54,55 @@ clone_script_template: &CLONE_SCRIPT_TEMPLATE
       git reset --hard $CIRRUS_CHANGE_IN_REPO
     fi
 
+trigger_gcp_pre_prod_deployer_release_task: &TRIGGER_GCP_PRE_PROD_DEPLOYER_RELEASE_TASK
+  # only_if: $TRIGGER == "PUSH_STAGING_DEPLOYER" || $BRANCH == "master"
+  ec2_instance:
+    <<: *VM_TEMPLATE
+  login_to_gcr_script:
+    - export DOCKER_GCLOUD_PASSWORD=$(echo ${DOCKER_GCLOUD_SA_KEY} | base64 -d)
+    - docker login -u _json_key -p "$DOCKER_GCLOUD_PASSWORD" https://${GCLOUD_REGISTRY}
+  install_helm_script:
+    - curl -fsSL -o get_helm.sh https://raw.githubusercontent.com/helm/helm/master/scripts/get-helm-3
+    - chmod 700 get_helm.sh
+    - ./get_helm.sh
```

Comment:
This could probably be cached with the new system, but considering how fast and small it is, lets consider it for an hardening ?

Reply:
I agree. I've defined a SONAR-21764 ticket for it.